[2022-04-13 22:33:20,939] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-02-02T06:00:10+00:00 [queued]>
[2022-04-13 22:33:21,026] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-02-02T06:00:10+00:00 [queued]>
[2022-04-13 22:33:21,031] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-04-13 22:33:21,037] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-04-13 22:33:21,047] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-04-13 22:33:21,145] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): ingest> on 2022-02-02 06:00:10+00:00
[2022-04-13 22:33:21,168] {standard_task_runner.py:52} INFO - Started process 10096 to run task
[2022-04-13 22:33:21,184] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'LocalIngestionDAG', 'ingest', 'scheduled__2022-02-02T06:00:10+00:00', '--job-id', '1680', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_local.py', '--cfg-path', '/tmp/tmpn6bk56un', '--error-file', '/tmp/tmp4mwikm2q']
[2022-04-13 22:33:21,193] {standard_task_runner.py:77} INFO - Job 1680: Subtask ingest
[2022-04-13 22:33:21,487] {logging_mixin.py:109} INFO - Running <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-02-02T06:00:10+00:00 [running]> on host 13db72185839
[2022-04-13 22:33:21,814] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=LocalIngestionDAG
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2022-02-02T06:00:10+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-02T06:00:10+00:00
[2022-04-13 22:33:21,824] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-04-13 22:33:21,883] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'ls /opt/***']
[2022-04-13 22:33:21,924] {subprocess.py:85} INFO - Output:
[2022-04-13 22:33:21,932] {subprocess.py:89} INFO - ***.cfg
[2022-04-13 22:33:21,936] {subprocess.py:89} INFO - dags
[2022-04-13 22:33:21,937] {subprocess.py:89} INFO - logs
[2022-04-13 22:33:21,937] {subprocess.py:89} INFO - output.csv
[2022-04-13 22:33:21,945] {subprocess.py:89} INFO - plugins
[2022-04-13 22:33:21,951] {subprocess.py:89} INFO - requirements.txt
[2022-04-13 22:33:21,954] {subprocess.py:89} INFO - scripts
[2022-04-13 22:33:21,955] {subprocess.py:89} INFO - webserver_config.py
[2022-04-13 22:33:21,956] {subprocess.py:93} INFO - Command exited with return code 0
[2022-04-13 22:33:22,081] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=LocalIngestionDAG, task_id=ingest, execution_date=20220202T060010, start_date=20220413T223320, end_date=20220413T223322
[2022-04-13 22:33:22,172] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-13 22:33:22,696] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
